app/brave_generated_resources.grd

```diff
@@ -891,6 +891,25 @@ Or change later at <ph name="SETTINGS_EXTENIONS_LINK">$2<ex>brave://settings/ext
         </message>
       </if>
 
+      <!-- Text recognition dialog-->
+      <if expr="enable_text_recognition">
+        <message name="IDS_CONTENT_CONTEXT_COPY_TEXT_FROM_IMAGE" desc="The name of the context menu item to get text from image">
+          Copy Text From Image
+        </message>
+        <message name="IDS_TEXT_RECOG_DIALOG_CLOSE_BUTTON" desc="The text for the close button of text recognition dialog">
+          Close
+        </message>
+        <message name="IDS_TEXT_RECOG_DIALOG_HEADER_IN_PROGRESS" desc="The text for the header of text recognition dialog">
+          Copying text from image...
+        </message>
+        <message name="IDS_TEXT_RECOG_DIALOG_HEADER_COMPLETE" desc="The text for the header of text recognition dialog">
+          Text copied from image
+        </message>
+        <message name="IDS_TEXT_RECOG_DIALOG_HEADER_FAILED" desc="The text for the header of text recognition dialog">
+          Text copy failed
```

Comment:
This is also shown when there is no text in the image. Is that OK? @rmcfadden3 @rebron

Reply:
@bsclifton — what would be the user action in that case? Would it be like they were trying to copy the image itself instead? If the feature is called "Text copy" then we might want to stick with "Text copy failed" regardless of whether user actually tried to copy text or not. Just for consistency.

I suppose alternatively, you could just do "Copy failed" instead, but that'd be Plan B for me. @rebron wdyt?

Reply:
@rmcfadden3 In this proposed use-case (copying a picture of a dog or something)- it didn't necessarily fail... there wasn't text in the image. `Text copy failed` might actually be perfect - since there wasn't any text. I think I was focusing on the `Failed` part too much

Reply:
@bsclifton — Ah, I see. I mean, if we can get that granular with the error trigger, we could say something like "No text to copy" or "Error: no text in this image to copy" or similar. But yeah, if we need to keep it general, "Text copy failed" would probly suffice.